client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -35,8 +35,28 @@ const BrewPage = (props)=>{
 		index    : 0,
 		...props
 	};
+
+	const pageRef = useRef();
+
+	const rootMargin = '-50% 0px';
+
+	useEffect(()=>{
+		const observer = new IntersectionObserver(
+			([entry])=>{
+				const pageNo = parseInt(entry.target.getAttribute('id').slice(1));
```

Comment:
I wonder if we can just use the component's `index` prop directly instead of reading it from the DOM target?